gfw_pixetl/utils/utils.py

```diff
@@ -28,6 +32,36 @@ def __init__(self, dst: Dict) -> None:
         self.metadata: Dict = {}
 
 
+def create_empty_file(work_dir, dst_profile: Dict[str, Any]):
+    # FIXME: This grabs the dtype and nodata value from dst_profile,
+    # but I think it should ACTUALLY copy those values from the profile
+    # of one of the input files in case they are different (for example
+    # if the source files are int32 and the desired output is uint16).
+    # It matters because there currently seems to be a problem mixing
+    # dtypes in one VRT.
+    local_file_path = os.path.join(work_dir, "input", "empty_file.tif")
+    profile = {
+        "driver": "GTiff",
+        "dtype": dst_profile["dtype"],
```

Comment:
This might cause an error if the input datatype is different from the output data type.